<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Historiador v0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_code">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>종목코드 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_start">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>시작날짜 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_end">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>종료날짜 :</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupbox_index">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>지표</string>
    </property>
    <widget class="QRadioButton" name="button_PBR">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PBR</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="button_PER">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PER</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_range">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>연도범위 :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_enter">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>그리기</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_code">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_start">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_end">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_range">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>891</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="chart"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
